scripts/env/dev_cconf_amd64.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+. "$SCRIPTS_DIR/env/common.sh"
+
+export PROJECT=${PROJECT:-"console-conf-tests"}
+export CHANNEL=${CHANNEL:-"beta"}
```

Comment:
Couldn't the default variables be defined on `scripts/env/common.sh`? There's a lot of boilerplate if not.